tests/cli/test_train_model.py

```diff
@@ -72,6 +72,32 @@ def test_train_explicit_validation_test(
     assert Path(output).is_file()
 
 
+def test_continue(monkeypatch, tmp_path):
+    """Test that continuing training from a checkpoint runs without an error raise."""
+    monkeypatch.chdir(tmp_path)
+    shutil.copy(RESOURCES_PATH / "qm9_reduced_100.xyz", "qm9_reduced_100.xyz")
+    shutil.copy(RESOURCES_PATH / "bpnn-model.pt", "bpnn-model.pt")
+    shutil.copy(RESOURCES_PATH / "options_continue.yaml", "options_continue.yaml")
+
+    command = ["metatensor-models", "train", "options_continue.yaml"]
```

Comment:
But this is not using the continue flag?

Reply:
Sorry, added it